source/ShipyardPanel.cpp

```diff
@@ -259,24 +259,34 @@ void ShipyardPanel::Buy(bool onlyOwned)
 
 
 
-bool ShipyardPanel::CanSell(bool toStorage) const
+ShopPanel::TransactionResult ShipyardPanel::CanSell() const
 {
 	return playerShip;
 }
 
 
 
-void ShipyardPanel::Sell(bool toStorage)
+void ShipyardPanel::Sell(bool storeOutfits)
 {
 	static const int MAX_LIST = 20;
 
 	int count = playerShips.size();
 	int initialCount = count;
-	string message;
-	if(!toStorage)
-		message = "Sell the ";
+	string message = "";
```

Comment:
containers are implicitly empty
```suggestion
	string message;
```